server/controllers/CommentController.js

```diff
@@ -63,6 +63,56 @@ class CommentController {
       }));
   }
 
+  /**
+   * @description This method add new comment on a comment
+     * @param {object} req
+     * @param {object} res
+     * @param {object} next
+     * @returns {object} returns an object of comment
+   */
+  static async addCommentToComment(req, res, next) {
+    const { id } = req.userData;
+    const { articleId, commentId } = req.params;
+    const { reply } = req.body;
+    let result;
+    let commenter;
+    try {
+      const comment = await Comment.findByPk(commentId);
+      if (comment.articleId !== +articleId) {
+        const err = new Error('this comment does not belong to this article');
+        err.status = 400;
+        return next(err);
+      }
```

Comment:
I don't think this check here is necessary either since we should be referencing the comment directly by it's unique id. If you remove the `articleId` param from the endpoint uri, you should remove this as well.

Reply:
Since I remove the articleId, this would not be necessary anymore.